build/jobs/cleanup.yml

```diff
@@ -54,3 +54,19 @@ jobs:
         Import-Module $(System.DefaultWorkingDirectory)/release/scripts/PowerShell/FhirServerRelease/FhirServerRelease.psd1
 
         Remove-AadTestAuthEnvironment -TestAuthEnvironmentPath $(System.DefaultWorkingDirectory)/testauthenvironment.json -EnvironmentName $(DeploymentEnvironmentName)
+
+- job: deleteImage
+  displayName: 'Delete Image'
+  pool:
+    vmImage: $(WindowsVmImage)
+  steps:
+  - task: AzureCLI@2
+    displayName: 'Delete Image'
+    imputs:
+      azureSubscription: $(ConnectedServiceName)
+      scriptType: ps
+      scriptLocation: InlineScript
+      inlineScript: |
+        az acr repository delete --name $(azureContainerRegistryName) --image 'r4_fhir-server:$(ImageTag)'
```

Comment:
So we just remove the image at the end of the build, makes sense

Reply:
I added this to help with clean up, but it isn't perfect as it only runs if the build succeeds. The other script will clean up any from failed builds.